### PR TITLE
Add PullApprove config

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

**Comment by Shredder121** [comment]
How about adding this one, so we can't merge `progress` PRs?
[docs](http://docs.pullapprove.com/yaml/always_pending/)
```yaml
always_pending:
  labels: progress
```

File: .pullaprove.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+approve_by_comment: true
+approve_regex: (?i)^Approved
```
Comment (Shredder121) [inline .pullaprove.yml]: Maybe something close to the one on the [docs website](http://docs.pullapprove.com/yaml/approve_regex/)?
```yaml
approve_regex: '(?i)^(approved|looks good|:shipit:|:\+1:)'
```
Then "Looks good to me." would for example match as well.

**Comment by Shredder121** [comment]
Can/will you also turn off [`author_approval`](http://docs.pullapprove.com/yaml/author_approval/)?
Then the dashboard looks better with the two lists (Your PRs, PRs requiring approval).

**Comment by Shredder121** [comment]
A little annoying that you can't test out the configuration.
I suppose we'll have to see what happens if it can't read the configuration correctly.

**Comment by Shredder121** [comment]
How many approvals shall we do? 2?
Or just 1, and preferably someone else merging it?
We don't want to lock up the repo when most of us are on vacation I suppose.

**Comment by timowest** [comment]
One approval is enough, but it needs to be someone else.

**Comment by Shredder121** [comment]
I won't go over the top with the regex, but what do you think of this:
```
(?i)^(approved|(this (also )?)?looks (good|fine|okay)|:shipit:|:\+1:)
```
This means that these would match:
https://github.com/querydsl/querydsl/pull/1816#issuecomment-203996318
https://github.com/querydsl/querydsl/pull/1883#issuecomment-219216111
https://github.com/querydsl/querydsl/pull/1883#issuecomment-219217235
https://github.com/querydsl/querydsl/pull/1804#issuecomment-216073271
https://github.com/querydsl/querydsl/pull/1900#issuecomment-226970951
https://github.com/querydsl/querydsl/pull/1900#issuecomment-229086241
https://github.com/querydsl/querydsl/pull/1820#issuecomment-201258515

The only few so far encountered that wouldn't match are somewhat like these:
https://github.com/querydsl/querydsl/pull/1841#issuecomment-216337601
https://github.com/querydsl/querydsl/pull/1836#issuecomment-203997474
But I'll just have to go easy on the responses.

**Comment by timowest** [comment]
I prefer the current state of the regex, let's use a consistent style

**Comment by Shredder121** [comment]
Fair enough.

**Comment by Shredder121** [comment]
It seems that the configuration in `master` is used, so I can not yet approve this.
Is there anything you want to change?

**Comment by timowest** [comment]
Looks good

**Comment by Shredder121** [comment]
I'm not _sure_ whether they already implemented the new issue webhook events (edits).
So I think, for now, you'll have to make a new comment.

**Comment by Shredder121** [comment]
Also, what does your 'merge pull request' button look like?
Is it grey, or red because you are an admin?

**Comment by timowest** [comment]
Approved

**Pull request merged**